prediction_market_agent/agents/known_outcome_agent/benchmark.py

```diff
@@ -0,0 +1,165 @@
+import time
+import typing as t
+from datetime import datetime, timedelta
+
+import pytz
+from dotenv import load_dotenv
+from prediction_market_agent_tooling.benchmark.agents import AbstractBenchmarkedAgent
+from prediction_market_agent_tooling.benchmark.benchmark import Benchmarker
+from prediction_market_agent_tooling.benchmark.utils import (
+    Market,
```

Comment:
Will release a new version of prediction-market-agent-tooling, and bump it in this repo and update this code. Happy to do as part of this PR or as a follow-up PR.